libbeat/processors/add_cloud_metadata/_meta/fields.yml

```diff
@@ -44,3 +49,14 @@
       type: alias
       path: cloud.region
       migration: true
+
+    - name: meta.cloud.account.id
```

Comment:
Do not have the full context on this one but I wonder why `migration: true` is missing?

Reply:
I submitted another comment with the review :innocent:, I guess we don't need these alias cause they are primary minded for backwards compatibility, but these are new fields

Reply:
I copied it from existing fields.yml

I though that those fields needed to be defined at the event, and then aliased to ECS.
are there some docs on how to use this fields.yml at processors and how it fits with ECS?

Reply:
Sorry I don't think there are docs about this. These aliases were created in the context of ECS transition (see https://github.com/elastic/beats/pull/9265 for more details). In a nutshell, old fields were not ECS compliant, so everything was moved to ECS fields, then old fields became aliases pointing to the new ones, as a way to allow smoother upgrades for users.

As you are adding new fields, there is no need to create alias from old names, as nobody is relying on these names for their visualizations